cps-v2/enrollments.go

```diff
@@ -0,0 +1,123 @@
+package cps
+
+import (
+  "github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid"
+  "github.com/akamai/AkamaiOPEN-edgegrid-golang/client-v1"
+)
+
+// Enrollments represents a CPS enrollments list
+type Enrollments struct {
+  Enrollments []Enrollment `json:"enrollments"`
+}
+
+// NewEnrollments creates an empty Enrollments
+func NewEnrollments() *Enrollments {
+  e := &Enrollments{}
+  return e
+}
+
+// Get populates Enrollments with the list of avaiable enrollments
+//
+// API Docs: https://developer.akamai.com/api/luna/cps/resources.html#getenrollments
+func (e *Enrollments) Get(config edgegrid.Config) error {
+  if req, err := client.NewRequest(
+    config,
+    "GET",
+    GET_ENROLLMENTS_URI,
```

Comment:
Given that this is tied to a specific version of the API, endpoints **should not** change; abstracting this out to a constant potentially obscures things like GET parameters, and I think it would be better to just hardcode in the URIs.